Automattic/wp-calypso
# Jetpack Dashboard: Create column for Stats

## Proposed Changes

* Create a new column for stats in the Jetpack Agency Pro Dashboard.

## Testing Instructions

* Run `git checkout add/stats-column-pro-dashboard` and `yarn start-jetpack-cloud`
* Open http://jetpack.cloud.localhost:3000/, and you'll be redirected to the /dashboard.
* Verify that the table design is as shown below -- the columns will show up when the feature flag `jetpack/pro-dashboard-expandable-block` is enabled.

![image](https://user-images.githubusercontent.com/2293077/224411350-88398c36-93c0-4bb4-b27d-eeb4c5bbcf3f.png)

![image](https://user-images.githubusercontent.com/2293077/224412056-fac38a19-5f05-4f1d-95a2-19b415027732.png)


## Pre-merge Checklist
